hack/update-depstat.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+kube::golang::setup_env
+
+
+echo 'installing depstat'
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  go install github.com/RinkiyaKeDad/dependency-analyzer-poc
```

Comment:
when you run this command, you will see the generated binary is `./_output/local/go/bin/dependency-analyzer-poc` (not `depstat` which you use below in line 23)